ee-feature-pack/channel/pom.xml

```diff
@@ -29,12 +35,12 @@
 
         <!-- We need the feature pack built first as that generates the manifest -->
         <dependency>
-            <groupId>${project.groupId}</groupId>
+            <groupId>org.wildfly</groupId>
             <artifactId>wildfly-ee-galleon-pack</artifactId>
+            <version>${project.version}</version>
```

Comment:
I don't think this should be needed as it's under dependencyManagement in the root pom.

If it is needed, please use ${ee.maven.version}.

Reply:
I think the same issue with groupId is causing problems here making the version necessary